### PR TITLE
Fix locale on Android

### LINKED ISSUE #86: setLocale method does not work (android)
Platform android:

```
cordova.plugins.instabug.setLocale(
  'german', 
  ok => console.log('succ', ok), 
  err => console.log('err', err)
);
```

Output:

`err Invalid action`

### PR DESCRIPTION
**Pull request by alyezz**
* Fixes #86

### CODE REVIEW

File: src/android/IBGPlugin.java [hunk 1]
```diff
@@ -1133,6 +1135,25 @@ public void setShouldShowSurveysWelcomeScreen(final CallbackContext callbackCont
...
+    /**
+     * Change Locale of Instabug UI elements(defaults to English)
+     *
+     *
+     * @param args .optString(0)      locale name
+     *
+     * @param callbackContext Used when calling back into JavaScript
+     */
+    public void setLocale(final CallbackContext callbackContext, JSONArray args) {
+        String locale = args.optString(0);
```
Comment (AliAbdelfattah) [inline src/android/IBGPlugin.java:1147]: Could args be passed as null? If it's possible, then I'd suggest moving it inside try catch block

Comment (alyezz) [reply src/android/IBGPlugin.java:1147]: The parent method already wraps everything in a try/catch

Comment (AliAbdelfattah) [reply src/android/IBGPlugin.java:1147]: buuut.. we already have a try catch block in here :D

Comment (alyezz) [reply src/android/IBGPlugin.java:1147]: I'm following the same implementation in the rest of the class, all methods have try/catch blocks inside them. Also other `parse` methods don't.

In any case i think we will have to refactor this in the future to match `argsRegistry` in RN and Flutter

**Pull request merged**